content/docs/16.faq/pebble.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Pebble FAQ
+---
+
+## What's Pebble ?
```

Comment:
```suggestion
## What is Pebble and why do we use it at Kestra?
```

Reply:
please fix the same in all titles etc like "How to reading outputs & inputs?"

you can either remove `?` or remove space